diff --git a/node_modules/roslib/src/core/Ros.js b/node_modules/roslib/src/core/Ros.js
index fa4ca73..2655b33 100644
--- a/node_modules/roslib/src/core/Ros.js
+++ b/node_modules/roslib/src/core/Ros.js
@@ -3,7 +3,8 @@
  * @author Brandon Alexander - baalexander@gmail.com
  */
 
-var WebSocket = require('ws');
+// This causes issues in modern browsers because WebSocket are already defined
+// var WebSocket = require('ws');
 var WorkerSocket = require('../util/workerSocket');
 var socketAdapter = require('./SocketAdapter.js');
 
diff --git a/node_modules/roslib/src/util/workerSocket.js b/node_modules/roslib/src/util/workerSocket.js
index d7d683f..049859b 100644
--- a/node_modules/roslib/src/util/workerSocket.js
+++ b/node_modules/roslib/src/util/workerSocket.js
@@ -1,8 +1,7 @@
-var work = require('webworkify');
 var workerSocketImpl = require('./workerSocketImpl');
 
 function WorkerSocket(uri) {
-  this.socket_ = work(workerSocketImpl);
+  this.socket_ = workerSocketImpl
 
   this.socket_.addEventListener('message', this.handleWorkerMessage_.bind(this));
 
diff --git a/node_modules/roslib/src/util/workerSocketImpl.js b/node_modules/roslib/src/util/workerSocketImpl.js
index 07c49de..1e7621f 100644
--- a/node_modules/roslib/src/util/workerSocketImpl.js
+++ b/node_modules/roslib/src/util/workerSocketImpl.js
@@ -1,5 +1,3 @@
-var WebSocket = WebSocket || require('ws');
-
 module.exports = function(self) {
   var socket = null;
 
